src/dropdown/dropdown.ts

```diff
@@ -93,12 +132,44 @@ export class NgbDropdown {
     }
   }
 
+  ngAfterContentInit() {
+    if (this.container === 'body') {
+      this._zoneSubscription = this._ngZone.onStable.subscribe(() => {
+        positionElements(
```

Comment:
Actually, this has even larger performance implications. If you put a console.log(1); inside the subscribe, you'll see they happen more and more until it crashes. I think too many subscribes are happening. Is this supposed to complete on the first result? It seems like they're building up endlessly. I had 50x 1, then 604x 1 then 80,000x 1 until the browser crashed